apps/scan/src/main/openapi/scan.yaml

```diff
@@ -3700,6 +3700,18 @@ components:
             `firstCollected` if the validator has collected in only one round
           type: integer
           format: int64
+        weight:
+          description: |
+            Weight of the activity records produced by this license. Determined
+            through SV votes. If not set, weight is 1.0
```

Comment:
wdyt about making the field required and always setting the default value. Thereby limiting the propagation of the default? No need for the clients to implement the defaulting, or is there?

Reply:
By always returning the default value, instead of missing field/`null`, we are losing information which may be useful for the clients. I can set it if required.

Reply:
I think we're better off not leaking that implementation detail.